exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/CryptoFunctions.java

```diff
@@ -0,0 +1,315 @@
+/**
```

Comment:
Please include Apache header in a form of comment. Please correct here and in files below.